File: app/src/main/java/com/example/android/tflitecamerademo/Home/HomeActivity.java
package com.example.android.tflitecamerademo.Home;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import com.example.android.tflitecamerademo.R;
import com.example.android.tflitecamerademo.RecognitionFirebase.RecognitionFirebaseActivity;
import com.example.android.tflitecamerademo.RecognitionObjects.CameraActivity;
import com.example.android.tflitecamerademo.RecognitionObjectsTensorFlow.Camera2Activity;


public class HomeActivity extends AppCompatActivity {

    CardView information, hand, object, objectFirebase;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

        information = findViewById(R.id.cardInformation);
        hand = findViewById(R.id.cardHand);
        object = findViewById(R.id.cardObject);
        objectFirebase = findViewById(R.id.cardObjectFirebase);

        information.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(HomeActivity.this, AboutPageDevelopers.class);
                startActivity(intent);
            }
        });

        hand.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(HomeActivity.this, Camera2Activity.class);
                startActivity(intent);
            }
        });

        object.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(HomeActivity.this, CameraActivity.class);
                startActivity(intent);
            }
        });

        objectFirebase.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(HomeActivity.this, RecognitionFirebaseActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/android/tflitecamerademo/Home/HomeActivity.kt
package com.example.android.tflitecamerademo.Home

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.android.tflitecamerademo.R
import com.example.android.tflitecamerademo.RecognitionFirebase.RecognitionFirebaseActivity
import com.example.android.tflitecamerademo.RecognitionObjects.CameraActivity
import com.example.android.tflitecamerademo.RecognitionObjectsTensorFlow.Camera2Activity
import kotlinx.android.synthetic.main.activity_home.*

class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)

        cardHand.setOnClickListener {
            val intent = Intent(this@HomeActivity, Camera2Activity::class.java)
            startActivity(intent)
        }
        cardObject.setOnClickListener {
            val intent = Intent(this@HomeActivity, CameraActivity::class.java)
            startActivity(intent)
        }
        cardObjectFirebase.setOnClickListener {
            val intent = Intent(this@HomeActivity, RecognitionFirebaseActivity::class.java)
            startActivity(intent)
        }
    }
}